### ISSUE TITLE
Possibility to make empty plots

### ISSUE THREAD

**Issue by HomoCodens** [mbannert/tstools#214]
Somewhat related to #213:
For the tsviewr package it would be neat if `tsplot` could also render an empty plot with just axes.
Simplest option would be an `empty` parameter that just skips the drawing calls?
Alternatively make it work with missing inputs (in which case the ranges must be specified of course).

What would be good defaults for such a plot?

Also, maybe think up a "default"/"representative"/signature plot (e.g. KOFBaro + ref + forecast with CI).

Like:
![MathWorks logo](https://ch.mathworks.com/etc/designs/mathworks/img/pic-header-mathworks-logo.svg)

Instantly recognizable.

**Comment by mbannert** [comment]
good idea. I agree empty plots would be really helpful.  Not sure 'bout icon capability, yet. Maybe we should go into that hex sticker direction, too.
